include/dolphin/card.h

```diff
@@ -159,6 +159,7 @@ typedef struct CARDID {
 #define CARD_RESULT_CANCELED     -14
 #define CARD_RESULT_FATAL_ERROR -128
 
+// TODO: This should be private
```

Comment:
what are you basing that on?

Reply:
The SDK doesn't seem to have those macros public, and the public API doesn't deal with blocks.